DotNet/DataAcquisition/Controllers/AuthenticationConfigController.cs

```diff
@@ -73,27 +73,42 @@ private async Task SendAudit(string message, string correlationId, string facili
         QueryConfigurationTypePathParameter queryConfigurationTypePathParameter,
         CancellationToken cancellationToken)
     {
-        if(string.IsNullOrWhiteSpace(facilityId))
-        {
-            return BadRequest();
-        }
-
         try
         {
+            if (queryConfigurationTypePathParameter == null)
```

Comment:
This is always going to be 'false' since it's an enum and it's not defined as nullable. The framework will throw a 500 if it's not provided.

Reply:
Okay - I made it nullable so that we have an opportunity to handle the case and provide information instead of throwing an uncontrolled 500